assets/js/modules/analytics-4/utils/report-validation.test.js

```diff
@@ -157,4 +158,135 @@ describe( 'Analytics 4 Reporting API validation', () => {
 			).toBe( false );
 		} );
 	} );
+
+	describe( 'isValidOrders', () => {
+		it( 'should return TRUE if an array of valid order objects is passed', () => {
```

Comment:
All these tests repeat the same logic. Let's group them into one table test using `it.each( ... )( ... )` approach.

Reply:
Sounds good :+1: